video_xblock/backends/brightcove.py

```diff
@@ -516,13 +517,26 @@ def get_default_transcripts(self, **kwargs):
 
     def download_default_transcript(self, url, language_code=None):  # pylint: disable=unused-argument
         """
-        Downloads default transcript from a video platform API in WebVVT format.
+        Download default transcript from a video platform API in WebVVT format.
 
         Arguments:
-            url (str): transcript download url.
+            url (str): Transcript download url.
         Returns:
-            unicode: Transcripts in WebVTT format.
+            sub (unicode): Transcripts formatted per WebVTT format https://w3c.github.io/webvtt/
         """
+
         data = requests.get(url)
-        sub = unicode(data.content.decode('utf8'))
+        text = data.content.decode('utf8')
+        # To clean subs text from special symbols here, we need `unescape()` from xml.sax.saxutils
+        # Reference: https://wiki.python.org/moin/EscapingHtml
+        html_unescape_table = {
+            "&amp;": "&", "&amp; ": "&", " &amp;": "&",
+            "&quot;": '"', "&quot; ": '"', " &quot": '"',
+            "&amp;#39;": "'", "&amp;#39; ": "'", " &amp;#39;": "'",
+            "&apos;": "'", "&apos; ": "'", " &apos;": "'",
+            "&gt;": ">", "&gt; ": ">", " &gt;": ">",
+            "&lt;": "<", "&lt; ": "<", " &lt;": "<",
```

Comment:
Otherwise, some special symbols may be left unescaped. I saw a case with  `"&amp;#39; "`, so I added spaces to other unescape symbols.